FirebaseRemoteConfig/Tests/SwiftAPI/APITests.swift

```diff
@@ -110,6 +105,41 @@ class APITests: APITestBase {
     waitForExpectations()
   }
 
+  // Test New API.
+  // Contrast with testChangedActivateWillNotFlag in FakeConsole.swift.
+  func testUnchangedActivateWillFlag() {
+    let expectation = self.expectation(description: #function)
+    config.fetch { status, error in
+      if let error = error {
+        XCTFail("Fetch Error \(error)")
+      }
+      XCTAssertEqual(status, RemoteConfigFetchStatus.success)
+      self.config.activate { changed, error in
+        XCTAssertTrue(!APITests.useFakeConfig || changed)
+        if let error = error {
+          print("Activate Error \(error)")
```

Comment:
should this use `XCTFail` like it does above?

Reply:
This new API should assert on error being nil. Good catch!